src/reporting/Core_json_output.ml

```diff
@@ -595,7 +595,7 @@ let core_output_of_matches_and_errors (res : Core_result.t) : OutJ.core_output =
          * but should we compute skipped and scanned here instead?
```

Comment:
What about this `TODO` note?

Reply:
I'm not totally sure what it means. There are two paths of output due to `osemgrep` and `pysemgrep`. This path is only related to `pysemgrep` output. I assume that we need to update here too. `osemgrep` output already has the `scanned` data.

Reply:
We need to at least update that TODO note or something, according to our best knowledge.

Reply:
done.